src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -155,10 +155,12 @@ void RegisterContainerAdapter(IContainer containerToAdapt)
             container.ConfigureComponent<IBuilder>(_ => b, DependencyLifecycle.SingleInstance);
         }
 
+
+        bool ShouldRunInstallers => settings.GetOrDefault<bool>("Installers.Enable");
```

Comment:
I'd prefer to see this be a local instead of a new field. It could either be passed in, or it could be used to conditionally call the methods.